### PR TITLE
OperatorHub sync

### PR DESCRIPTION
**Pull request by vmuzikar**
This PR is just to sync the changes that were recently made in the OperatorHub. Namely:
* 13.0.1 release
   * Related PRs:
      * https://github.com/operator-framework/community-operators/pull/3926
      * https://github.com/operator-framework/community-operators/pull/3927
* Migration to bundle format ([KEYCLOAK-16128](https://issues.redhat.com/browse/KEYCLOAK-16128))
   * Related PRs:
      * https://github.com/operator-framework/community-operators/pull/4168
      * https://github.com/operator-framework/community-operators/pull/4176)
* 14.0.0 release
   * Related PRs:
      * https://github.com/operator-framework/community-operators/pull/4177
      * https://github.com/operator-framework/community-operators/pull/4178

### CODE REVIEW

**Comment by vmuzikar** [comment]
The test failures are unrelated and fixed in #359.

File: deploy/olm-catalog/keycloak-operator/ci.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ---
 # Use `replaces-mode` or `semver-mode`. Once you switch to `semver-mode`, there is no easy way back.
-updateGraph: replaces-mode
\ No newline at end of file
+updateGraph: replaces-mode
+reviewers:
+  - slaskawi
```
Comment (abstractj) [inline deploy/olm-catalog/keycloak-operator/ci.yaml:5]: @vmuzikar I believe it's safe to remove Seba from the list of reviewers. If you would like to, please include me as the second reviewer.

Comment (vmuzikar) [reply deploy/olm-catalog/keycloak-operator/ci.yaml:5]: @abstractj This is from the times of 13.0.0 when he was still around. I'll create PR to the OperatorHub then to update the reviewers and I'll update this PR afterwards.

Comment (slaskawi) [reply deploy/olm-catalog/keycloak-operator/ci.yaml:5]: I'm still here! :)

Comment (abstractj) [reply deploy/olm-catalog/keycloak-operator/ci.yaml:5]: @slaskawi yes, you are and I appreciate that. But we need to update the list of maintainers, considering your limited time.

**Review by abstractj — DISMISSED** [review]

**Comment by abstractj** [comment]
@vmuzikar approving it regardless.

File: deploy/olm-catalog/keycloak-operator/ci.yaml (hunk 1 above)
Comment (vmuzikar) [reply deploy/olm-catalog/keycloak-operator/ci.yaml]: @abstractj Updated the reviewers. Related PRs:
* https://github.com/operator-framework/community-operators/pull/4229
* https://github.com/operator-framework/community-operators/pull/4230

**Pull request merged**